### PR TITLE
Add a warning about limited support for NGINX

### PR DESCRIPTION
**Pull request by acsfer**
Warn user about limited support to webservers other than Apache.

### CODE REVIEW

File: admin_manual/installation/nginx.rst [hunk 1]
```diff
@@ -1,10 +1,14 @@
 ===================
-Nginx configuration
+NGINX configuration
 ===================
 
-This page covers example Nginx configurations to use with running a Nextcloud
-server. These configurations examples were originally provided by
-`@josh4trunks <https://github.com/josh4trunks>`_ and are community-maintained. (Thank you contributors!)
+.. warning::
```
Comment (josh4trunks) [inline admin_manual/installation/nginx.rst:5]: I'm just wondering why we have `.. subtitle::`?
I'm not familiar enough with RST format to know what that is trying to achieve?

**Review by josh4trunks — APPROVED** [review]
I just have 1 question, but it isn't really substantive. Change makes sense to me.

**Review by MorrisJobke — APPROVED** [review]
Thanks 👍

File: admin_manual/installation/nginx.rst (hunk 1 above)
Comment (MorrisJobke) [reply admin_manual/installation/nginx.rst:5]: I don't see the subtitle in here. Or are you referring to another part of that document?

**Pull request merged**